HealthcareSystem/GraphicalEditor/Models/MapObjectRelated/MapObjectType.cs

```diff
@@ -159,5 +174,24 @@ public List<String> AllMapObjectTypes
                 return allMapObjectTypes;
             }
         }
+
+        public List<String> AllMapObjectTypesAvailableToChange
+        {
+            get
+            {
+                List<String> allMapObjectTypesAvailableToChange = new List<String>();
+                Array typesOfMapObjects = Enum.GetValues(typeof(TypeOfMapObject));
+                foreach (TypeOfMapObject enumValue in typesOfMapObjects)
+                {
+                    if (enumValue != TypeOfMapObject.BUILDING && enumValue != TypeOfMapObject.ROAD)
+                    {
+                        MapObjectType value = new MapObjectType(enumValue);
+                        allMapObjectTypesAvailableToChange.Add(value.ObjectTypeFullName);
+                    }
+                }
```

Comment:
### Comment

We have a problem with the descriptive variable name, **enumValue** and **value** is not enough descriptive.

### Proposal

Algorithm for better naming
• Describe the thing in a sentence or two (easier to do in Serbian, then translate).
• Remove any words that are not crucial to convey the meaning.
• Remove any words that can be inferred from the Type, containing function, containing class, or containing module.
• See if what remains sufficiently describes the thing.
• Rename.

Reply:
@kmilena98 could you solve this?

Reply:
done